### DISCUSSION TITLE
Remember position in playlist?

### DISCUSSION THREAD

**Discussion by ministryofsillywalks** [music-assistant/support#3247] [Q&A] [answered]
Before I go off trying to figure this out in my own I thought I'd ask if someone already implemented this.

My kids like to listen to audiobooks on Spotify. Longer books are often split up into separate albums with several tracks.
I made playlists out of these but now I would love to save the position where they stopped last time.

My first thought was to get the track number in the playlist and save this in some helper and then when the playlist is called again start from the track number stored in the helper.

Has someone implemented something similar or has any better ideas?

Or is a feature like this coming with Audiobook and podcast support soonish and I shouldn't bother building some janky fix? ;)

**Comment by ministryofsillywalks** [comment]
Just found
https://github.com/orgs/music-assistant/discussions/3050
Looks like starting at a certain track in a playlist/album isn't doable via automation?

**Reply by OzGav** [reply]
It is too early to speculate on what functionality might be available when the audiobook/podcast support is added but saving where you are up to isn't an out there idea. You will just need to be patient as we aren't seeing any significant dev assistance that can kick this along.

**Reply by ministryofsillywalks** [reply]
No worries.
But did I interpret the other discussion correctly, that I currently can't use a service to start a playlist at position X ?

**Reply by OzGav** [reply, marked as answer]
Correct

**Comment by ministryofsillywalks** [comment]
Not feasible at the moment

**Comment by Tsamsiyu1** [comment]
Is this doable now?

**Reply by OzGav** [reply]
Audiobooks do resume where they left off.

**Reply by Tsamsiyu1** [reply]
yeah but can you do this with an audiobook series.  I have for example a script for spotify that plays a playlist of  audiobooks at evening to go to bed. it is running 30 minutes and then stoped but it oly advances about 10 minutes every day.  when the audiobook is finished it plays thenext in this series, because it is in the playlist.

**Reply by OzGav** [reply]
MA will remember where in the QUEUE it is up to but if the queue is replaced on the player then that is lost. You could use an automation to hold the current item I would expect?

**Reply by Tsamsiyu1** [reply]
i just found the api it looks like i can do it via the API.
load a playlist, skip to my position saved in my db, seek to position and play.
API is doable but would be nice if this will someday come to the home assistant integration as well. thanks for the help

EDIT: Just for Info it works great with http requests.
